sycl/doc/extensions/EnqueueBarrier/enqueue_barrier.asciidoc

```diff
@@ -1,4 +1,4 @@
-= SYCL_INTEL_enqueue_barrier
+= SYCL_EXT_INTEL_ENQUEUE_BARRIER
```

Comment:
I wonder if this should be `SYCL_EXT_ONEAPI_ENQUEUE_BARRIER` instead.

I may be wrong, but I don't think the functionality here is intended to be specific to Intel hardware.  OpenCL has had `clEnqueueBarrier` for a long time (see [here](https://www.khronos.org/registry/OpenCL/sdk/1.1/docs/man/xhtml/clEnqueueBarrier.html)).

Reply:
You are right, thanks, fixed in https://github.com/intel/llvm/pull/4432/commits/8047c6ff82bb4519e2e8cbda1f817526ee5e915a